src/installer/_core.py

```diff
@@ -0,0 +1,113 @@
+"""Wheel Installation logic, sans I/O
+"""
+
+import posixpath
+
+from installer._compat import StringIO
+from installer._compat.typing import TYPE_CHECKING
+from installer.parsers import parse_metadata_file
+from installer.protocols import Destination, Scheme, WheelSource
+from installer.validation import InvalidWheelSource
+
+if TYPE_CHECKING:
+    from typing import Callable, List, Optional
+
+    from installer._compat.typing import Text
+
+    Validator = Callable[[WheelSource], None]
+
+
+__all__ = ["Installer"]
+
+
+class Installer(object):
+    """Implements wheel installation logic
+
+    Supports Wheel version 1.0 (PEP 427).
+    """
+
+    def __init__(self, name, validators):
+        # type: (Optional[Text], List[Validator]) -> None
+        super(Installer, self).__init__()
+        self.name = name
+        self._validators = validators
+
+    def install(self, source, destination, requested=False):
```

Comment:
What is "requested"? This could be interpreted as meaning, "someone explicitly requested the install". But this seems to be more about the `REQUESTED` metadata.

I would also say that should be a keyword-only parameter when Python 3-only support lands.

Reply:
That interpretation is correct, since that's what the REQUESTED file represents.

I haven't added any details to this docstring yet, so I'll document the parameter when I get to this.

OK to make it a keyword-only parameter (will document so in the initial version as well).